### PR TITLE
Upgrade kubecube-monitoring helm chart

### PR DESCRIPTION
**Pull request by just1900**
1. upgrade kubecube-monitoring helm chart
2. add pre-fetch images

### CODE REVIEW

File: images/v1.20.9/images.list [hunk 1]
```diff
@@ -13,6 +13,8 @@ hub.c.163.com/kubecube/kube-state-metrics:v1.9.8
 hub.c.163.com/kubecube/prometheus-operator:v0.47.0
 hub.c.163.com/kubecube/node-exporter:v1.1.2
 hub.c.163.com/kubecube/prometheus:v2.26.0
```
Comment (weilaaa) [inline images/v1.20.9/images.list:15]: hub.c.163.com/kubecube/prometheus:v2.26.0 --> hub.c.163.com/kubecube/prometheus:v2.26.1 ?

**Review by weilaaa — COMMENTED** [review]
please confirm this image

File: images/v1.20.9/images.list (hunk 1 above)
Comment (just1900) [reply images/v1.20.9/images.list]: yup, changed

**Review by weilaaa — APPROVED** [review]
LGTM

**Pull request merged**